src/domain/safe/entities/schemas/multisig-transaction-type.schema.ts

```diff
@@ -0,0 +1,50 @@
+import { Schema } from 'ajv';
+import { confirmationSchema } from './multisig-transaction.schema';
+
+export const multisigTransactionTypeSchema: Schema = {
+  type: 'object',
+  properties: {
+    txType: { type: 'string', const: 'MULTISIG_TRANSACTION' },
+    safe: { type: 'string' },
+    to: { type: 'string' },
+    value: { type: 'string', nullable: true },
+    data: { type: 'string', nullable: true },
+    dataDecoded: { type: 'object', nullable: true },
```

Comment:
Should we use the existent schema? 🤔 
```javascript
dataDecoded: dataDecodedSchema,
```

Reply:
Yes we should 👍

Reply:
[1a34acc](https://github.com/5afe/safe-client-gateway-nest/pull/170/commits/1a34acc0a7af949761b2b34a22ac35b73c951e41)